packages/vue/README.md

```diff
@@ -0,0 +1,125 @@
+[La version française suit.](#composants-de-système-de-design-gc--react)
+
+# GC Design System Components – Vue
+
+GC Design System Components – Vue provides the [GC Design System Components](https://github.com/cds-snc/gcds-components/tree/main/packages/web) within a Vue wrapper to ease integration of the components into Vue apps.
+
+## Documentation
+
+You can find the full documentation for GC Design System Components on [https://design-system.alpha.canada.ca/](https://design-system.alpha.canada.ca/).
+
+## Installation
+
+### Install from npm
+
+Navigate to the root folder of your project and run:
+
+``` js
+npm install @cdssnc/gcds-components-vue
+```
+
+In your `main.js` file, import the GC Design System components plugin and use it:
+
+``` jsx
+import { GcdsComponents } from '@cdssnc/gcds-components-vue';
+
+createApp(App).use(GcdsComponents).mount('#app');
+```
+
+In your `App.vue`, place the following to load the global styles:
+
+``` html
+<style src='@cdssnc/gcds-components-vue/gcds.css'>
+    /* global styles */
+</style> 
+```
+
+### Using Vite
+
+If using [Vite](https://vitejs.dev/), additional configuration will need to be added to `vite.config.ts` to prevent Vue from logging a warning about failing to resolve components.
```

Comment:
```suggestion
If using [Vite](https://vitejs.dev/), additional configuration will need to be added to `vite.config.ts` to prevent Vue from logging a warning about failing to resolve components e.g. "Failed to resolve component: my-component").
```
could we add the example stencil gives in their docs too?